### DISCUSSION TITLE
Disable organization users (non administrators) to create new organizations by setting

### DISCUSSION THREAD

**Discussion by hanrea** [zitadel/zitadel#5235] [Ideas]
In theory, the relationship between organizations is independent. New organizations can be created by instance administrators or use apply form.

**Comment by hanrea** [comment]
I don't quite understand the purpose of the instance? And difference between organization and instance?

**Reply by fforootd** [reply]
You might want to check this here https://zitadel.com/docs/concepts/structure/instance and https://zitadel.com/docs/concepts/structure/organizations

Simply speaking all organizations are in an instance. The instance holds the api endpoints, signing keys and so on. Organizations contain on the other hand users, projects and so on.

**Reply by hanrea** [reply]
I mean, the relationship between instances and organizations is a bit vague. 
Currently, users are unique in the organization, and not in the instance. But the custom domain is bound to the instance. The instance is like a company entity for own users. Users should belong to this instance rather than the organization.

In some case, users should be associated with domain names, which is closer to the mainstream situation.
Since the open source version of zitadel cannot be used as a saas service similar to zitadel.cloud, the concept of instance should be removed to reduce the overhead.

**Reply by fforootd** [reply]
Ah got it. All users must be unique in an instance. However we Suffix the username with the org domains.

You can disable the setting "Add organization domain as suffix to loginnames" on the instance domain settings.

This allows you to use any username in any org.

**Reply by hanrea** [reply]
I'm wrong. The two users I mentioned are two users with the same email. Their user names cannot be the same. But now I find that this 
Email can't log in normally (tip:  no user can be found)

**Comment by eliobischof** [comment]
> In theory, the relationship between organizations is independent. New organizations can be created by instance administrators or use apply form.

@hanrea by reading the comment above and the discussion title, am I right that you are also addressing the issue #4986?

**Reply by hanrea** [reply]
@eliobischof  Yes ,But my discussion is to determine the design goal of zitadel. I hope to find a service that can provide saas authentication instead of independent user authentication software.
It seems that the domain name is bound to the instance in the source code. I will continue to track the development of zitadel .
